VideoWeb/VideoWeb/ClientApp/src/sass/abstracts/_breakpoints.scss

```diff
@@ -0,0 +1,20 @@
+@import 'functions';
+
+$breakpoints: (
+  smallest: 0px,
+  extra-small : 480px,
+  small: 641px,
+  medium : 768px,
+  large: 1024px,
+);
+
+// 1024px
+$breakpoint-large: em(map-get($breakpoints, large));
+// 1023px
+$breakpoint-medium-max: em(map-get($breakpoints, large)-1px);
```

Comment:
Hi Farron, is this one still needed

Reply:
Actually no. The only case where this was used was in the example file.